app/scripts/views/form.js

```diff
@@ -57,7 +60,7 @@ define(function (require, exports, module) {
       BaseView.call(this, options);
 
       // attach events of the descendent view and this view.
-      this.delegateEvents(_.extend({}, FormView.prototype.events, this.events));
+      this.delegateEvents(_.extend({}, proto.events, this.events));
```

Comment:
Apologies if stupid question, but does it matter that the second arg has changed from `FormView.prototype` to `BaseView.prototype`?